fleet-provisioning/src/entry.c

```diff
@@ -28,6 +28,8 @@
 #define MAX_TEMPLATE_PARAM_LEN 4096
 #define MAX_PATH_LEN 4096
 
+#define USER_GROUP GGL_SYSTEMD_SYSTEM_USER ":" GGL_SYSTEMD_SYSTEM_GROUP
```

Comment:
```suggestion
#define USER_GROUP ( GGL_SYSTEMD_SYSTEM_USER ":" GGL_SYSTEMD_SYSTEM_GROUP )
```

This probably fixes clang-tidy's warning